### PR TITLE
✨ 940 - integrate ego redirect

### PR DESCRIPTION
**Pull request by anncatton**
**Description of changes**

Removes redirect paths from localStorage for regular login path and specified path
For a request with specified path, will use redirect changes from https://github.com/overture-stack/ego/releases/tag/2.9.1 
Does NOT include changes for join flow, that will be done in a separate PR.
Search params in the redirect url are encoded so google oauth is happy

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/submission-system/SubmissionSystemLockedNotification.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const SubmissionSystemLockedNotification = ({
   };
 
   return (
-    (isWorksapceDisabled && showNotification && (
+    (isWorkspaceDisabled && showNotification && (
```
Comment (hlminh2000) [inline components/pages/submission-system/SubmissionSystemLockedNotification.tsx]: ??? what changed here?

File: uikit/Button/GoogleLogin/index.tsx [hunk 2]
```diff
@@ -21,10 +21,10 @@ const GoogleLogin: React.ComponentType<{
   className?: string;
   redirectPath?: string;
   onClick?: React.MouseEventHandler<HTMLAnchorElement>;
-}> = ({ link, id, className, onClick }) => {
+}> = ({ link, id, className, onClick, redirectPath = '' }) => {
   const theme = useTheme();
   return (
-    <StyledLink id={id} href={link} className={className} onClick={onClick}>
+    <StyledLink id={id} href={`${link}${redirectPath}`} className={className} onClick={onClick}>
```
Comment (hlminh2000) [inline uikit/Button/GoogleLogin/index.tsx]: safer to use `url-join` as env vars are not reliable :P. make sure to add it to `uikit/package.release.json` too :P

File: components/pages/submission-system/SubmissionSystemLockedNotification.tsx (hunk 1 above)
Comment (anncatton) [reply components/pages/submission-system/SubmissionSystemLockedNotification.tsx]: just a typo fix!

Comment (hlminh2000) [reply components/pages/submission-system/SubmissionSystemLockedNotification.tsx]: ah!!! thank you :D lol that's so trippy

Comment (anncatton) [reply components/pages/submission-system/SubmissionSystemLockedNotification.tsx]: ikr?!

**Pull request merged**